model/src/main/kotlin/utils/DependencyGraphConverter.kt

```diff
@@ -27,13 +27,15 @@ import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.PackageLinkage
 import org.ossreviewtoolkit.model.PackageReference
 import org.ossreviewtoolkit.model.Project
+import org.ossreviewtoolkit.model.config.Excludes
```

Comment:
Maybe link to https://github.com/oss-review-toolkit/ort/issues/3825 from the commit message for reference?